src/component/admin/users/UsersList/UsersList.tsx

```diff
@@ -49,8 +57,14 @@ const UsersList = () => {
     const [inviteLink, setInviteLink] = useState('');
     const [delUser, setDelUser] = useState<IUser>();
     const ref = useLoading(loading);
+    const { filtered, setFilter } = useUsersFilter(users);
+    const { sorted, sort, setSort } = useUsersSort(filtered);
     const { page, pages, nextPage, prevPage, setPageIndex, pageIndex } =
-        usePagination(users, 50);
+        usePagination(sorted, 50);
```

Comment:
I believe what you have done here should work, but can you verify that reducing the number of users per page and sorting also affects the pagination UI?

Reply:
Quickly tested with `usePagination(sorted, 2)` and everything seems to be behaving correctly, both for sorting and searching.